shumai/tensor/tensor_ops.ts

```diff
@@ -36,3 +36,15 @@ export function thresholdRelu(tensor: Tensor, threshold = 1): Tensor {
   const mask = tensor.gte(scalar(threshold))
   return tensor.mul(mask)
 }
+
+export function clamp(tensor: Tensor, low: number, high: number): Tensor {
+  return tensor.clip(full(tensor.shape, low), full(tensor.shape, high));
```

Comment:
I wasn't aware `clip` did not work with scalars -- not sure why that is. `clamp` might be a little redundant but kept for consistency with FL.